bazel/grpc_build_system.bzl

```diff
@@ -108,7 +108,7 @@ def grpc_proto_library(name, srcs = [], deps = [], well_known_protos = False,
     generate_mock = generate_mock,
   )
 
-def grpc_cc_test(name, srcs = [], deps = [], external_deps = [], args = [], data = [], uses_polling = True, language = "C++"):
+def grpc_cc_test(name, srcs = [], deps = [], external_deps = [], args = [], data = [], uses_polling = True, language = "C++", size = "medium"):
```

Comment:
it seems "small" may be a better default. From the docs:
> Unittests are considered "small", integration tests "medium", and end-to-end tests "large" or "enormous". Blaze uses the size to determine a default timeout (which can be overridden using the "timeout" attribute) and the amount of resources that have to be acquired for the test to run. The mapping of test size to resource is currently:

Reply:
I used medium as the default value because because that's what Bazel doc says, so that's the size used by all tests currently. I wonder what it actually implies if the stated size is larger than the actual one?

Reply:
Using timeout instead